klayout/drc/README.md

```diff
@@ -0,0 +1,93 @@
+# DRC Documentation
+
+Explains how to use the runset.
+
+## Folder Structure
+
+```text
+📦drc
+ ┣ 📦testing
+ ┣ 📦rule_decks
+ ┣ 📜README.md
+ ┗ 📜run_drc.py
+ ```
+
+## Rule Deck Usage
+The `run_drc.py` script takes a gds file to run DRC rule decks of GF180 technology with switches to select subsets of all checks. 
+
+### Requirements
+Please make sure to use the latest Klayout setup at your side.
```

Comment:
link to https://www.klayout.de/build.html?

Reply:
Done